src/openrct2-ui/interface/FileBrowser.cpp

```diff
@@ -174,7 +174,7 @@ namespace OpenRCT2::Ui::FileBrowser
                 Guard::Fail("Unsupported load/save directory type.");
         }
 
-        return "";
+        return nullptr;
```

Comment:
Does the code that calls this actually check for `nullptr`? Otherwise, we’ll have a crash where there wasn’t one before.